src/pages/study-room/components/timer.tsx

```diff
@@ -1,15 +1,47 @@
 import { Button, Card, CardActions, CardContent, CardHeader } from '@mui/material';
+import { useMemo } from 'react';
+import useTimer from '../hook/useTimer';
+
+type KeyType = { [key in string]: string };
+const STEP_TEXT: KeyType = {
+  PLANNING: '계획',
+  STUDYING: '스터디',
+  RETROSPECT: '회고',
+  RESTING: '휴식',
+  COMPLETED: '완료',
+};
+
 const Timer = () => {
+  /** api호출
+   *
+   * const { isLoading, isError, data, error, refetch } = useQuery({
+   * queryKey: ['get-study-room-info'],
+   * queryFn: () => studyRoomInfo({ studyId: 1 }),
+   * });
+   *
+   */
+
+  const { seconds, startTimer, status } = useTimer({
+    updateAt: '2024-03-11T14:55:32.513Z',
+    step: 'PLANNING',
+  });
+
+  const currentTime = useMemo(() => {
+    const min = Math.floor(seconds / 60);
+    const sec = seconds % 60;
```

Comment:
개~~~인적으로는 요런식으로 표현해줘도 좋다고 생각이 들긴 하네요!
```suggestion
    const 분 = Math.floor(seconds / 60);
    const 초 = seconds % 60;
    
    return `${String(분).padStart(2, '0'):${String(초).padStart(2, '0')}`;
```